private/pkg/thread/thread.go

```diff
@@ -87,16 +87,19 @@ func Parallelize(ctx context.Context, jobs []func(context.Context) error, option
 		select {
 		case <-ctx.Done():
 			stop = true
+			retErr = multierr.Append(retErr, fmt.Errorf("jobs[%d]: context canceled", jobIdx))
```

Comment:
This should likely be `ctx.Err()` instead of a custom error, some people parse the specific context canceled function